Realm/RLMUtil.mm

```diff
@@ -280,7 +280,8 @@ static inline BOOL RLMIsSubclass(Class class1, Class class2) {
 }
 
 BOOL RLMIsObjectSubclass(Class klass) {
-    return RLMIsSubclass(class_getSuperclass(klass), RLMObjectBase.class);
+    static Class objectBaseClass = RLMObjectBase.class;
```

Comment:
A function-level static with a nontrivial initializer has the equivalent of a `dispatch_once()` wrapping the initialization. That's significantly cheaper than `NSClassFromString` (for the RealmSwift stuff), but probably not cheaper than a simple message send.